uber/tests/test_migration.py

```diff
@@ -0,0 +1,6 @@
+from uber.tests import dump_alembic_schema, dump_reset_uber_db_schema
+
+
+def test_alembic_migrations():
+    assert dump_alembic_schema() == dump_reset_uber_db_schema(), (
+        'Missing or invalid migrations, see alembic/README.md')
```

Comment:
Nice!  That also answers my question above about why we want to still support ``reset_uber_db()``.